ipapython/ipaldap.py

```diff
@@ -1405,12 +1411,15 @@ def find_entries(self, filter=None, attrs_list=None, base_dn=None,
 
             while True:
                 if paged_search:
-                    sctrls = [SimplePagedResultsControl(0, page_size, cookie)]
+                    sctrls.append(
+                        SimplePagedResultsControl(0, page_size, cookie))
+                if get_effective_rights:
```

Comment:
That doesn't look right. I think you have to create a new ``sctrls`` list in every loop. Otherwise the controls are going to pile up while looping over results.

Reply:
You're right.  Good catch.